### PR TITLE
Added dev workspaces operator

### PR DESCRIPTION
**Pull request by james-cccc**
Added required manifests for the dev workspaces (previously crw) operator

### CODE REVIEW

File: openshift-dev-spaces/instance/base/checluster.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+apiVersion: org.eclipse.che/v1
+kind: CheCluster
+metadata:
+  name: devspaces
+  namespace: openshift-operators
```
Comment (pittar) [inline openshift-dev-spaces/instance/base/checluster.yaml:5]: @james-cccc - is this the recommended namespace to install the CheCluster operator?  It does seem to be the default, but it doesn't seem right to actually deploy this into "openshift-operators".  If it is the correct place, then let me know and I'll merge the PR as-is.

Comment (james-cccc) [reply openshift-dev-spaces/instance/base/checluster.yaml:5]: Agreed, let me look into this and test it a little

Comment (pittar) [reply openshift-dev-spaces/instance/base/checluster.yaml:5]: I asked over in the Eclipse Che room, but I haven't had a response yet.  In 2.0 I think the default was "openshift-workspaces".  If there's no "official" namespace, perhaps "openshift-devspaces" would be a good option.

Comment (james-cccc) [reply openshift-dev-spaces/instance/base/checluster.yaml]: Yeah exactly, 'openshift-workspaces' used to be it. Now we're not creating any additional namespaces so I've opted for a 'devspaces' namespace for the moment for an example.

Comment (pittar) [reply openshift-dev-spaces/instance/base/checluster.yaml]: Just in time for Mario Loriedo to confirm that the default should be `openshift-devspaces`.  Do you mind making one more update, please? :)

Comment (james-cccc) [reply openshift-dev-spaces/instance/base/checluster.yaml]: Ohh that's interesting, so installing the latest available version of the operator on a 4.10 cluster does not create any additional namespaces like 'openshift-devspaces' and of course manually creating any 'openshift-*' namespace doesn't fly. hmm... will have a further look into this

Comment (pittar) [reply openshift-dev-spaces/instance/base/checluster.yaml]: If you're interested in the explanation:
https://issues.redhat.com/browse/CRW-2997

So, the operator does indeed get installed in `openshift-operators`, but the `CheCluster` is supposed to be deployed into `openshift-devspaces` - although that bit of docs seems to be missing.

Comment (james-cccc) [reply openshift-dev-spaces/instance/base/checluster.yaml]: Thanks for the info, much appreciated. Set it up to be openshift-devspaces

**Comment by pittar** [comment]
Looks good.  Thanks!

**Pull request merged**